listenbrainz/db/couchdb.py

```diff
@@ -129,6 +129,37 @@ def fetch_data(prefix: str, user_id: int):
 
     return None
 
+def get(database: str, document_id: str):
```

Comment:
All this will be removed before merging, yes?

Reply:
Yes

Reply:
Done.